lib/bot_framework/message.rb

```diff
@@ -0,0 +1,92 @@
+module BotFramework
+  class Message
+    TEXT_FORMAT = {
+      plain: 'plain',
+      markdown: 'markdown',
+      xml: 'xml'
+    }
+    ATTACHMENT_LAYOUT = {
+      list: 'list',
+      carousel: 'carousel'
+    }
+
+    def initialize(session = nil)
+      @data = {}
+      @data[:type] = 'consts.MessageType' #FIXME
+      @data[:agent] = 'consts.agent'
+      if @session
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.